src/Form/Form.js

```diff
@@ -151,8 +152,11 @@ class Form {
                 formValues.credentials.username = probableField.value
             } else {
                 // If we still don't have a username, try scanning the form's text for an email address
-                this.form.querySelectorAll('*:not(select):not(option)').forEach(el => {
+                this.form.querySelectorAll('*:not(select):not(option)').forEach((el) => {
                     const elText = getText(el)
+                    // Ignore long texts to avoid false positives
+                    if (elText.length > 70) return
```

Comment:
I noticed this somewhere that we were saving something like privacy@site.com as the username. There was a big privacy policy text. This will prevent such occurrences.

Reply:
This seems like a sensible limit. Just confirming, would setting it to something like 256 (the max length of an email) end up catching privacy policy text?

Reply:
This is an edge case of an edge case, so I would prefer avoiding false positives for everyone rather than trying to catch the rare instances where someone with a mega email incurs in this specific edge case.